### ISSUE TITLE
Task "success" not found

### ISSUE THREAD

**Issue by garak** [deployphp/deployer#2244]
### Describe the bug

When I try to run `dep deploy production`, it fails immediately with exception mentioned in title.
I don't have a "success" task, nor I can found any task with such name inside deployer source.

### To reproduce

Use following `deploy.php`:

```php
<?php
namespace Deployer;

require 'recipe/symfony.php';

host('production')
    ->setHostname('myhost')
    ->set('deploy_path', '~/{{application}}')
    ->set('user', 'myuser')
;
```
run `dep deploy production`

### Expected behavior

A deploy, or at least an error that you can understand

### Environment

- Deployer version: 7.0.0-beta.3
- PHP version: 7.4.11
- Deployment target(s) OS: Ubuntu 20.04

This is my stack trace:

```
  at phar:///myproject/vendor/deployer/dist/dep/src/Task/TaskCollection.php:20
 Deployer\Task\TaskCollection->throwNotFound() at phar:///myproject/vendor/deployer/dist/dep/src/Collection/Collection.php:30
 Deployer\Collection\Collection->get() at phar:///myproject/vendor/deployer/dist/dep/src/Task/ScriptManager.php:62
 Deployer\Task\ScriptManager->doGetTasks() at n/a:n/a
 array_map() at phar:///myproject/vendor/deployer/dist/dep/src/Task/ScriptManager.php:81
 Deployer\Task\ScriptManager->doGetTasks() at phar:///myproject/vendor/deployer/dist/dep/src/Task/ScriptManager.php:32
 Deployer\Task\ScriptManager->getTasks() at phar:///myproject/vendor/deployer/dist/dep/src/Command/MainCommand.php:103
 Deployer\Command\MainCommand->execute() at phar:///myproject/vendor/deployer/dist/dep/vendor/symfony/console/Command/Command.php:258
 Symfony\Component\Console\Command\Command->run() at phar:///myproject/vendor/deployer/dist/dep/vendor/symfony/console/Application.php:920
 Symfony\Component\Console\Application->doRunCommand() at phar:///myproject/vendor/deployer/dist/dep/vendor/symfony/console/Application.php:266
 Symfony\Component\Console\Application->doRun() at phar:///myproject/vendor/deployer/dist/dep/vendor/symfony/console/Application.php:142
 Symfony\Component\Console\Application->run() at phar:///myproject/vendor/deployer/dist/dep/src/Deployer.php:302
 Deployer\Deployer::run() at phar:///myproject/vendor/deployer/dist/dep/bin/dep:103
 require() at /myproject/vendor/deployer/dist/dep:4

```

**Comment by garak** [comment]
I found the definition of "success" task inside common recipe.
So, if I copy suck task in my deploy.php it works, but I think it should work without copying, since symfony recipe is including common recipe...

**Comment by antonmedv** [comment]
It should be replaced with deploy:success.

**Comment by garak** [comment]
> It should be replaced with deploy:success.

Replaced where?

**Comment by antonmedv** [comment]
Fixed via 5c736c07d4077f4422033a29f238c48d91efd3f1
